model/host/db.go

```diff
@@ -904,7 +907,7 @@ func FindUserDataSpawnHostsProvisioning() ([]Host, error) {
 	return hosts, nil
 }
 
-// Removes host intents that have been been uninitialized for more than 3
+// Removes host intents that have been been initializing for more than 3
```

Comment:
Remove duplicate `been`.

Reply:
Also, can you update this documentation to reflect the changes you made removing the condition of checking hosts that have been building for 15+ minutes?

Reply:
Done.